### PR TITLE
init model

### PR DESCRIPTION
**Pull request by tuannv007**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,40 +1,7 @@
...
-.idea/workspace.xml
-
-# Keystore files
-*.jks
+.gradle
+/local.properties
+.DS_Store
+/build
+/captures
+/.idea/
```
Comment (DoanVanToan) [inline .gitignore]: missing end of file

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        <activity android:name=".MainActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+    </application>
+
+</manifest>
```
Comment (DoanVanToan) [inline app/src/main/AndroidManifest.xml]: missing end of file

File: app/src/main/java/com/framgia/englishforkid_3/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package com.framgia.englishforkid_3;
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/englishforkid_3/MainActivity.java]: Chia package theo convention https://github.com/framgia/coding-standards/blob/master/eng/android/standard.md

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/activity_main"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingBottom="@dimen/activity_vertical_margin"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/activity_main.xml]: bh đặt tên dimen như sau dp_<value>, sp_<value>, color_<color_name>

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    android:paddingTop="@dimen/activity_vertical_margin"
+    tools:context="com.framgia.englishforkid_3.MainActivity">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Hello World!" />
```
Comment (DoanVanToan) [inline app/src/main/res/layout/activity_main.xml]: hardcode

File: app/src/test/java/com/framgia/englishforkid_3/ExampleUnitTest.java [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
+    @Test
+    public void addition_isCorrect() throws Exception {
+        assertEquals(4, 2 + 2);
+    }
+}
```
Comment (DoanVanToan) [inline app/src/test/java/com/framgia/englishforkid_3/ExampleUnitTest.java]: missing end of file

File: app/src/main/java/com/framgia/englishforkid_3/MainActivity.java (hunk 3 above)
Comment (tuannv007) [inline app/src/main/java/com/framgia/englishforkid_3/MainActivity.java]: em đã chia theo convention rồi ạ

File: app/src/main/res/values-w820dp/dimens.xml [hunk 7]
```diff
@@ -0,0 +1,6 @@
+<resources>
+    <!-- Example customization of dimensions originally defined in res/values/dimens.xml
+         (such as screen margins) for screens with more than 820dp of available width. This
+         would include 7" and 10" devices in landscape (~960dp and ~1280dp respectively). -->
+    <dimen name="activity_horizontal_margin">64dp</dimen>
```
Comment (DoanVanToan) [inline app/src/main/res/values-w820dp/dimens.xml]: Chưa sửa tên dimen

**Comment by vtabk2** [comment]
@DoanVanToan chỉ bạn ý cách format code + convention chưa?

**Comment by DoanVanToan** [comment]
@vtabk2 Ok rồi  a

**Comment by vtabk2** [comment]
ok @DoanVanToan

**Pull request merged**